pooranjoyb/popShop
# Adding an eye button

Summary of the Changes and the Related Issue:

This pull request implements a feature to toggle the visibility of the password in a signup form. When the user is typing, the password will be displayed, and it will be hidden once the user stops typing unless the eye icon is clicked to show it.

Motivation and Context:

The motivation behind this change is to enhance user experience by allowing users to see their password while typing. This is particularly useful for ensuring that users enter their passwords correctly. The context of this change is a React-based signup form where password visibility can be toggled with an eye icon.


Fixes # (issue)
In the current implementation of the signup form, the password input field does not provide an option for users to toggle the visibility of their password. This can lead to user frustration, as they cannot see what they are typing and may accidentally enter incorrect passwords. The objective is to implement a feature that allows users to show and hide their password by clicking an eye icon, which enhances usability and ensures that users can verify their password input easily.

## Type of change

Please give a X on it which is applicable

- [ ] Others(mentioned in the issue number)

# How Has This Been Tested?

**Test A**
    1. Navigate to the signup form.
    2. Type in the password field and observe that the password is visible while typing.
    3. Click the eye icon to toggle the password visibility on and off.
    4. Verify that the password hides when clicking the eye-off icon and shows when clicking the eye icon.

# Screenshorts and Vedios:
[Screencast from 30-05-24 07:43:28 PM IST.webm](https://github.com/pooranjoyb/popShop/assets/114069208/d4d20aa2-31fc-404c-8415-837ceec3a5e3)




# Checklist:
give a X on it which is applicable
